Bottom: d9c2e699f1e1ba4e8594d75b2371252c9669f3ca
Top:    2d2e369c79e203c61fa95e4f41a60f1b67974144
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-06-28 16:03:54 +0800

Refresh of virtio-net-switch-to-use-new

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index 54f33ce..cf6f969 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -1036,7 +1036,7 @@ static int virtnet_receive(struct receive_queue *rq, int budget)
 	void *buf;
 	struct virtnet_stats *stats = this_cpu_ptr(vi->stats);
 
-	if (vi->mergeable_rx_bufs) {
+	if (!vi->big_packets) {
 		void *ctx;
 
 		while (received < budget &&
